pkg/operator/options/options.go

```diff
@@ -39,8 +39,10 @@ var (
 type optionsKey struct{}
 
 type FeatureGates struct {
-	Drift    bool
 	inputStr string
+
+	Drift             bool
+	SpotConsolidation bool
```

Comment:
Should we call this SpotToSpot? 
```suggestion
	SpotToSpotConsolidation bool
```

Reply:
I have got this flag name already confirmed by Alex. I think spot consolidation naming is fine given that we do not support consolidating spot today ?

Reply:
> given that we do not support consolidating spot today

+1 to changing the name here, it's more lengthy but also more accurate. We actually _do_ support Spot consolidation today, if it's a delete of a spot node or the spot node is getting consolidated into an on-demand node.

Reply:
Changed this.